lib/semantic_puppet.rb

```diff
@@ -0,0 +1,9 @@
+require 'gettext-setup'
+
+module SemanticPuppet
+  GettextSetup.initialize(File.absolute_path('../locales', File.dirname(__FILE__)))
```

Comment:
Yeah, I was thinking we'd do a roundup on all the gettext related work in a separate PR, but in this PR would also work if anyone feels strongly.